great_expectations/data_context/store/tuple_store_backend.py

```diff
@@ -1151,11 +1151,9 @@ def list_keys(self, prefix: Tuple = ()) -> List[Tuple]:
 
     def get_url_for_key(self, key, protocol=None):
         az_blob_key = self._convert_key_to_filepath(key)
-        az_blob_path = os.path.join(  # noqa: PTH118
-            self.container, self.prefix, az_blob_key
-        )
+        az_blob_path = os.path.join(self.prefix, az_blob_key)  # noqa: PTH118
 
-        return "https://{}.blob.core.windows.net/{}".format(
+        return "https://{}.z16.web.core.windows.net/{}".format(
```

Comment:
Good question. That actually could change depending on the zone your storage lands in. I will look into expanding this PR to fetch a value for that.

Reply:
After some research I have found that the static website endpoint is not in any of the SDK classes but you would need to call the management api for it and I dont really want to require people putting Subscription ID and Resource group info just to get that done.
I'm thinking if an user could provide the baseurl for their docs site in the `store_backend` configs. That way the users could use their own domains to point to the data docs

Reply:
I think you are on to something with the baseurl. Maybe you can pass the `base_public_path` into the `TupleAzureBlobStoreBackend`. Then override `get_public_url_for_key` which is used when generating the URL (see SlackNotificationAction which calls context.get_docs_sites_urls() -> site_builder.get_resource_url() -> html_site_store.get_url_for_resource() which eventually calls the store `get_public_url_for_key()` method). I'm concerned that if we change `get_url_for_key()` that we are changing how all blobs are accessed, rather than changing the link displayed in the slack notification. Does that make sense? Take a look at how `get_public_url_for_key()` is used in some of the other stores.

Reply:
Thanks for the input. I will look into executing this next week.